pkg/descheduler/evictions/evictions.go

```diff
@@ -115,6 +122,11 @@ func (pe *PodEvictor) EvictPod(ctx context.Context, pod *v1.Pod, node *v1.Node,
 		return false, fmt.Errorf("Maximum number %v of evicted pods per %q node reached", *pe.maxPodsToEvictPerNode, node.Name)
 	}
 
+	if pe.maxPodsToEvictPerNamespace != nil && pe.nspodCount[pod.Namespace]+1 > *pe.maxPodsToEvictPerNamespace {
+		metrics.PodsEvicted.With(map[string]string{"result": "maximum number reached", "strategy": strategy, "namespace": pod.Namespace}).Inc()
```

Comment:
Can you update the result string in both cases to reflect the node/namespace case? E.g.
`metrics.PodsEvicted.With(map[string]string{"result": "maximum number of pods per namespace reached", "strategy": strategy, "namespace": pod.Namespace, "maximum": *pe.maxPodsToEvictPerNamespace}).Inc()`. Similar for the `maxPodsToEvictPerNode` case.

Reply:
Thanks a lot~  I will update soon :)